### DISCUSSION TITLE
v6.5.0

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible/ansible-lint#2312] [General]
## Minor Changes

- Add ability to process multi-document YAML files (#2290) @ssbarnea
- Standardize task MatchError enrichment (#2277) @cognifloyd
- Add name[casing] to identify wrongly capitalized task names (#2274) @ssbarnea
- Rename var-spacing rule to jinja[spacing] (#2259) @ssbarnea

## Bugfixes

- Reimplement jinja[spacing] to avoid use of regex (#2306) @ssbarnea
- Allow use of jinja with no_log (#2311) @ssbarnea
- Avoid erroring with empty vars files (#2310) @ssbarnea
- Avoid early of splitting shorthand string (#2308) @ssbarnea
- Remove pytest from default dependencies (#2307) @ssbarnea
- Fix ansible-lint pre-commit hook (#2303) @ssbarnea
- Change meta rules to use matchyaml instead of matchplay (#2296) @ssbarnea
- Make ruamel.yaml annotation implicit for yaml (#2299) @ssbarnea
- Prevent other rules from running on invalid yaml files (#2287) @ssbarnea
- Prevent duplicate warnings about deprecated tags (#2293) @ssbarnea
- Allow single letter role names (#2292) @ssbarnea
- Prevent occasional FileNotFoundError when removing the .lock file (#2291) @ssbarnea
- Standardize task MatchError enrichment (#2277) @cognifloyd
- Ensure rules are checked in specific order (#2284) @ssbarnea
- Prevent finding playbooks inside tasks folders (#2278) @ssbarnea
- Replacing os.system with subprocess functions (#2273) @jpodivin
- Reimplement unamed-task rule as name[missing] (#2263) @ssbarnea
- Implement file locking on cache directory (#2270) @ssbarnea
- Improve errors with invalid galaxy.yml files (#2262) @ssbarnea
- Fix: Progressive mode has no effect (#2261) @kostyaplis

<hr /><em>This discussion was created from the release <a href='https://github.com/ansible/ansible-lint/releases/tag/v6.5.0'>v6.5.0</a>.</em>

**Comment by netsandbox** [comment]
The documentation for the new `name[casing]` rule says that this applies to tasks and plays:

> This rule identifies several problems related to naming of tasks and plays.

but it actually also applies to handlers (defined in playbooks and roles).
Is this the intended behavior, then the documentation should be updated.
If not, then handlers should be skipped by this rule.

**Reply by netsandbox** [reply]
@ssbarnea any news regarding this?
